packages/es-components/src/components/containers/heading/Heading.tsx

```diff
@@ -1,23 +1,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled, { css } from 'styled-components';
+import styled, {
+  DefaultTheme,
+  ThemedStyledProps,
+  css
+} from 'styled-components';
+import { HeadingLevel } from '../../../../../../shared/types/src/ESTheme';
```

Comment:
We have an alias for this.

```suggestion
import { HeadingLevel } from 'es-components-shared-types';
```